bears/python/YapfBear.py

```diff
@@ -0,0 +1,137 @@
+            indent_dictionary_value: bool=False,
+            join_multiple_lines: bool=True,
+            spaces_around_power_operator: bool=False,
+            spaces_before_comment: int=2,
+            space_between_ending_comma_and_closing_bracket: bool=True,
+            split_arguments_when_comma_terminated: bool=False,
+            split_before_bitwise_operator: bool=False,
+            split_before_first_argument: bool=False,
+            split_before_logical_operator: bool=False,
+            split_before_named_assigns: bool=True,
+            use_spaces: bool=True,
+            based_on_style: str='pep8'):
+        """
+        :param max_line_length:
+            Maximum number of characters for a line.
+        :param tab_width:
+            Number of spaces per indent level.
+        :param allow_multiline_lambdas:
+            Allows lambdas to be formatted on more than one line.
+        :param blank_line_before_nested_class_or_def:
+            Inserts a blank line before a ``def`` or ``class`` immediately
+            nested within another ``def`` or ``class``.
+        :param coalesce_brackets:
+            Does not allow splitting consecutive brackets. Only relevant when
+            ``dedent_closing_brackets`` is set.
+        :param continuation_tab_width:
+            Indent width used for line continuations.
+        :param dedent_closing_brackets:
+            Puts closing brackets on a separate line, dedented, if the bracketed
+            expression can't fit in a single line. Applies to all kinds of
+            brackets, inceluding function definitions and calls.
+        :param indent_dictionary_value:
+            Indent the dictionary value if it cannot fit on the same line as
+            the dictionary key.
+        :param join_multiple_lines:
+            Join short lines into one line.
+        :param spaces_around_power_operator:
+            Set to ``True`` to prefer using spaces around ``**``.
+        :param spaces_before_comment:
+            The number of spaces required before a trailing comment.
+        :param space_between_ending_comma_and_closing_bracket:
+            Inserts a space between the ending comma and closing bracket of a
+            list, etc.
+        :param split_arguments_when_comma_terminated:
+            Splits before arguments if the argument list is terminated by a
+            comma.
+        :param split_before_bitwise_operator:
+            Set to ``True`` to prefer splitting before ``&``, ``|`` or ``^``
+            rather than after.
+        :param split_before_first_argument:
+            If an argument / parameter list is going to be split, then split
+            before the first argument.
+        :param split_before_logical_operator:
+            Set to ``True`` to prefer splitting before ``and`` or ``or`` rather
+            than after.
+        :param split_before_named_assigns:
+            Splits named assignments into individual lines.
+        :param use_tabs:
```

Comment:
ugh you have ``use_spaces`` in the argument list, and here you documented ``use_tabs``?? (line 43)